.github/workflows/ci.yml

```diff
@@ -70,10 +94,14 @@ jobs:
         run: sudo xcode-select -switch /Applications/Xcode_10.3.app
       - name: Build Carthage
         run: carthage build --archive
-      - name: Set Xcode version to 11.6
-        run: sudo xcode-select -switch /Applications/Xcode_11.6.app
+      - name: Set Xcode version to 11.7
+        run: sudo xcode-select -switch /Applications/Xcode_11.7.app
       - name: Build Carthage
         run: carthage build --archive
+      - name: Set Xcode version to 12.1
+        run: sudo xcode-select -switch /Applications/Xcode_12.1.app
+      - name: Build Carthage
+        run: ./carthage.sh build --archive
```

Comment:
For Xcode 12.x, there is a workaround in order to build.

See https://github.com/Carthage/Carthage/issues/3019#issuecomment-665136323